client/my-sites/checkout/upsell-nudge/plan-upgrade-upsell/index.jsx

```diff
@@ -54,30 +54,21 @@ export class PlanUpgradeUpsell extends PureComponent {
 		return (
 			<header className="plan-upgrade-upsell__small-header">
 				<h2 className="plan-upgrade-upsell__title">
-					{ translate( 'Hold tight, your site is being upgraded.' ) }
+					{ translate( 'This is a one time offer just for you' ) }
 				</h2>
 			</header>
 		);
 	}
 
 	body() {
 		const { translate, planRawPrice, planDiscountedRawPrice, currencyCode } = this.props;
-		const bundleValue = planRawPrice * 77;
-		const premiumThemePriceLow = planRawPrice * 0.73;
-		const premiumThemePriceHigh = planRawPrice * 1.045;
 		return (
 			<>
 				<h2 className="plan-upgrade-upsell__header">
 					{ translate(
-						'Add {{u}}%(bundleValue)s worth{{/u}} of Premium designs to your order {{br/}}{{u}}for just %(discountPrice)s more{{/u}}!',
+						'Upgrade your account to our most powerful plan ever {{br/}} with this special offer',
 						{
-							args: {
-								bundleValue: formatCurrency( bundleValue, currencyCode, { precision: 0 } ),
-								discountPrice: formatCurrency( planDiscountedRawPrice, currencyCode, {
```

Comment:
We could probably remove planDiscountRawPrice from [here too](https://github.com/Automattic/wp-calypso/blob/master/client/my-sites/checkout/upsell-nudge/index.jsx#L161).

Reply:
I might be missing something here, but this is the annual plan price with credits applied which is used in the copy [here](https://github.com/Automattic/wp-calypso/pull/43199/files/4339f6defad76364d168d1904e1a79cbce2eacff#diff-b70cd644cd92ff09bac429aa8cce9249R196-R205). Let me know if I'm overseeing something and this is not really needed!